cpp/cmake_modules/BuildUtils.cmake

```diff
@@ -239,8 +239,7 @@ function(ADD_ARROW_LIB LIB_NAME)
       PRIVATE_INCLUDES
       DEPENDENCIES
       SHARED_INSTALL_INTERFACE_LIBS
-      STATIC_INSTALL_INTERFACE_LIBS
-      OUTPUT_PATH)
```

Comment:
Hmm, did you mean to remove this parameter?

Reply:
Restored.